babel.config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": "94"
```

Comment:
What does this mean? How/when should this be updated in the future? How does this affect Firefox users?

Reply:
as we are not using babel to actually build anything... it doesn't matter... changed it to `{}` file needs to be there or eslint can't use the `@babel/eslint-parser`